core/src/main/java/com/datastrato/gravitino/connector/BaseCatalogPropertiesMetadata.java

```diff
@@ -48,6 +48,13 @@ public abstract class BaseCatalogPropertiesMetadata extends BasePropertiesMetada
                   null,
                   false,
                   false),
+              PropertyEntry.stringImmutablePropertyEntry(
+                  BaseCatalog.AUTHORIZATION_PROVIDER,
+                  "The classname of custom authorization provider",
```

Comment:
I don't think it is a class name, it is a provider name, the description is not accurate.

Reply:
Fixed.